pkg/apiserver/statement/statement.go

```diff
@@ -73,11 +77,29 @@ func (s *Service) schemasHandler(c *gin.Context) {
 func (s *Service) timeRangesHandler(c *gin.Context) {
 	db := utils.GetTiDBConnection(c)
 	timeRanges, err := QueryTimeRanges(db)
+	timeRangeResponse := []*TimeRange{}
+	for _, timeRange := range timeRanges {
+		beginTime, err := time.Parse(layout, timeRange.BeginTime)
```

Comment:
I think we should store timestamp as well. The human readable time will have problems with time zone.